nemo_curator/utils/distributed_utils.py

```diff
@@ -186,6 +187,8 @@ def read_single_partition(
         files: The path to the jsonl files to read.
         backend: The backend to use for reading the data. Either "cudf" or "pandas".
         add_filename: Whether to add a "filename" column to the DataFrame.
+        input-meta: A dictionary with the json object field names and data types.
```

Comment:
It would be good if this parameter could be a `Union[str, dict]` so that users could directly pass in a dictionary if they are using the Python API, or use the `str` if they are on the command line. Please update the docstring when you do this too to mention that the string should be "a string representation of a dictionary" or something along those lines.

Reply:
Done!